.travis.yml

```diff
@@ -26,14 +26,14 @@ install:
     - pip install numpy 'six>=1.13'
     - pip install traits
     - pip install argparse
-    - pip install pycrypto
+    - pip install pycryptodome
     - pip install pyyaml
     # module queue
     - pip install future
     - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then pip install subprocess32; fi
     - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then pip install importlib; fi
     - pip install codecov
-    - if [[ $TRAVIS_OS_NAME == linux ]]; then pip install pre-commit; fi
+    - if [[ $TRAVIS_OS_NAME == linux ]]; then pip install pre-commit~=1.0; fi
```

Comment:
Why force this ancient version of `pre-commit`? The casa-dev-5.0.3 image has pre-commit 2.13.0, I don't think that we have ever used version 1.x.
Moreover, we are stopping support for Python 2 in the master branch, and support for Python 3.5 is already dead. The AppVeyor and Travis configurations are badly outdated for this project, we should rewrite them completely.

Reply:
I do not have vision on all the projects, but I only find use of `crypt` in CatiDB projects, if no other project use `soma.crypt`, I agree to try to remove it.
When I created the PR, Travis gets an error, and the cause was pre-commits which wanted python>=3.6, so I thought downgrade to a v1.* would fix to build, but I saw that builds have been temporarily disabled for public repositories due to a negative credit balance. So maybe the modification is useless.

Reply:
Yes I think it would be better to revert this change, to avoid confusion (we do not want to support pre-commit < 2). At some point someone (maybe me) will take care of revamping the whole testing infrastructure for soma-base...